web/src/stream_edit.js

```diff
@@ -286,6 +287,29 @@ export function update_muting_rendering(sub) {
     $edit_container.find(".mute-note").toggleClass("hide-mute-note", !sub.is_muted);
 }
 
+function stream_notification_reset(e) {
+    const sub = get_sub_for_target(e.target);
+    if (!sub) {
+        blueslip.error("undefined sub in stream_notification_reset()");
+        return;
+    }
+    const data = [{stream_id: sub.stream_id, property: "is_muted", value: false}];
+    for (const [setting, option] of Object.entries(
```

Comment:
`setting`/`option` are bad names here; we want `per_stream_setting_name`, `global_setting_name`